packages/eos-components/src/components/ButtonsBar.vue

```diff
@@ -4,7 +4,7 @@
       v-for="button in buttonsVisible"
       :key="button"
       pill
-      size="sm"
+      size="md"
```

Comment:
We already went though a polish session and decided to make the buttons small. Let's ask Joana to confirm this change.